src/basic/boot/FStar.BigInt.fsti

```diff
@@ -53,9 +53,6 @@ val sqrt_big_int : (bigint -> bigint)
 val string_of_big_int : (bigint -> string)
 val big_int_of_string : (string -> bigint)
 
-val of_int : (int -> bigint)
-val to_int : (bigint -> int)
```

Comment:
These are type incorrect in F*; their implementation coerces between OCaml ints and bigint.